keywords.txt

```diff
@@ -0,0 +1,40 @@
+alert
```

Comment:
do you want to keep this file in top-level?

Reply:
well `olsconfig.yaml` is loaded from path specified in `OLS_CONFIG_FILE` (which might be different from root level). I'm just worried about having ton of files in top-level, some of them devel config, some as docs, other as required work files.